src/components/structures/LoggedInView.tsx

```diff
@@ -72,7 +72,7 @@ import LegacyCommunityPreview from "./LegacyCommunityPreview";
 // NB. this is just for server notices rather than pinned messages in general.
 const MAX_PINNED_NOTICES_PER_ROOM = 2;
 
-export function getInputableElement(el: HTMLElement): HTMLElement | null {
```

Comment:
No longer needed as an export since we use `checkInputtableElement(el)` instead.

`getInputableElement` does something slightly different and returns `HTMLElement` so we can't just consolidate this one away as well.

Reply:
So this used to look like `checkInputtableElement` recently until Germain (iirc) changed it, iirc because of how our Composer works where your caret might be within a paragraph/font/div/whatever within the contenteditable rather than directly and thus not working. Might be good to get a comment on why to use one vs the other.